src/components/containers/Sidebar/Sidebar.js

```diff
@@ -1,12 +1,31 @@
 import React, { Component } from 'react';
 
 import './Sidebar.css';
+import github from '../../../images/github.png';
+import info from '../../../images/blueinfo.png';
+import youtube from '../../../images/youtube.png';
+import bluedocumentation from '../../../images/bluedocumentation.png';
+import purple from '../../../images/Purple.png';
+
 
 class Sidebar extends Component {
   render() {
     return (
       <div className='sidebar'>
-        <p>This is a sidebar</p>
+        <div className="IconSizes">
```

Comment:
Any chance you could add an extra tab to everything inside this div? It just makes it easier to edit later.

Reply:
Checked